CMakeLists.txt

```diff
@@ -7,9 +7,49 @@ endif()
 
 project(binkstub VERSION 1.0.0 LANGUAGES C)
 
+include(CMakePackageConfigHelpers)
+include(GNUInstallDirs)
+
 add_library(binkstub SHARED bink.c)
+add_library(BinkStub::BinkStub ALIAS binkstub)
+if(NOT TARGET Bink::Bink)
+    add_library(Bink::Bink ALIAS binkstub)
+endif()
+if(WIN32 AND CMAKE_SIZEOF_VOID_P)
+    if(MINGW)
+        target_sources(binkstub PRIVATE bink_mingw.def)
+    endif()
+endif()
+set_target_properties(binkstub PROPERTIES DEFINE_SYMBOL "BINKDLL")
```

Comment:
Isn't this better done with `target_compile_definitions`?

Reply:
This renames the default `binkstub_EXPORTS`.

The result will be the same.

Reply:
Are you sure we are talking about the same line? This looks like it defines BINKDLL for the target, but is this property only set for the build or is it propagated to targets than link to binkstub?